bin/latest-stable

```diff
@@ -9,10 +9,16 @@
 # finally take last one
 ASDF_ELIXIR_LATEST_VERSION=$(./list-all | tr " " "\n" | grep -Ev "^0|^main|^master|otp|rc|^$" | tail -n 1)
 
+ASDF_VERSION="$(asdf version)"
 # gets current erlang version without headers
 # changes spaces to new line
 # takes second line (Name Version Source Installed)
-ASDF_ELIXIR_LATEST_OTP=$(asdf current --no-header erlang | tr -s " ." "\n" | sed -n 2p)
+if printf "0.16.0\n%s\n" "${ASDF_VERSION#v}" | sort -CV; then
```

Comment:
`sort -V` is was something the asdf maintainers determined wasn't supported everywhere. https://github.com/asdf-vm/asdf/blob/master/test/banned_commands.bats#L35-L37 I think it's best to consider another approach here. Otherwise this looks good to me.

If we can split on `.` then we can get major and minor versions from `ASDF_VERSION`. Then we can do if `$major < 1 && $minor < 16` for the old API.

Reply:
@Stratus3D Eh, `Bash` is such a mess … I don't get why everyone can support such trivial switch just for a compatibility reasons … Will take a look at it soon. Thanks for letting me know.